### DISCUSSION TITLE
Setup JWT access tokens signed with HS256

### DISCUSSION THREAD

**Discussion by yaskoo** [ory/hydra#2988] [Q&A] [answered]
I'm using hydra 1.10.6 and I'm trying to setup JWT access tokens signed using HS256.
I have set the `strategies.access_token` configuration to `jwt` and hydra is now issuing JWT access tokens, but I can't make it to sign them with HS256.
I've tried creating the client using the http api and providing the `jwks` key, but still tokens are signed with RS256.
Any pointers will be greatly appreciated.

btw, I've read [the docs](https://www.ory.sh/hydra/docs/advanced#json-web-tokens) and some [best practices](https://blog.bitsrc.io/best-practices-for-using-jwt-df3788433fd3) on JWTs - I know this is not a great combo, but it's something I have to live with for a while.

**Comment by aeneasr** [comment, marked as answer]
Hydra does not support that, it only supports asymmetric keys as otherwise anyone with the secret could forge tokens!

**Reply by yaskoo** [reply]
Thanks, that's good to know.
Is there a note somewhere in the docs (didn't see any). Maybe we should add one?

**Reply by aeneasr** [reply]
I don't think there is one, we can probably add one here:

https://github.com/ory/docs/blob/master/docs/hydra/jwks.md

would you be up for this?

**Reply by yaskoo** [reply]
sure, here it is https://github.com/ory/docs/pull/578
but im having trouble signing the CLA - I had to change my email in the commit and now the agreement doesn't show

**Comment by rm116** [comment]
Hello, 

If I want to use my private/public keys, is that possible ?
